src/test/resources/META-INF/persistence.xml

```diff
@@ -156,6 +156,8 @@
 		<!-- Profile package. -->
 		<class>org.sigmah.server.domain.profile.GlobalPermission</class>
 		<class>org.sigmah.server.domain.profile.OrgUnitProfile</class>
+		<class>org.sigmah.server.domain.profile.MainOrgUnitProfile</class>
+		<class>org.sigmah.server.domain.profile.SecondaryOrgUnitProfile</class>
```

Comment:
It seems we have a problem here: those two classes does not exist. I guess these have been added at one point in the development process but removed later and someone forget to remove these two lines while removing the classes themselves.

@ftavin, as the author of the commit, can you confirm? (and commit a fix)